### DISCUSSION TITLE
[@types/node] constant `SSL_OP_NO_RENEGOTIATION` is missing.

### DISCUSSION THREAD

**Discussion by RopoMen** [DefinitelyTyped/DefinitelyTyped#68182] [Issues with a @types package] [answered]
Constant `SSL_OP_NO_RENEGOTIATION` is missing from `crypto.d.ts`

Docs: https://nodejs.org/api/crypto.html#openssl-options

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment, marked as answer]
@RopoMen I have created the PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68204
